modules/dqt/jsx/components/button.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Why does the DQT need a separate button element than the LORIS ButtonElement?

Reply:
@driusan did this for a reason I'm having trouble recalling. The reason might have been the styling of the other button component in Loris not having an option for removing css that's for a button typically in a form like the majority of Loris. I'll take a look after meeting maybe I just wanted to make the button component class a function component.

Reply:
@driusan button is gone with latest commit